### ISSUE TITLE
TE3 crashing when copy-pasting column

### ISSUE THREAD

**Issue by MSmerdel** [TabularEditor/TabularEditor3#95]
I have a specific model in which TE3 crashes every time I try to copy a column or measure within a table (ctrl+c, ctrl+v).
It adds the new column and then crashes.
I can't figure out why this happens. I have tested it with and without the workspace database. It crashes in both scenarios.
The model is in compatibility level 1400.

In the Event Log the following exception is thrown:
Faulting application name: TabularEditor3.exe, version: 3.0.7838.17724, time stamp: 0x60cafeda
Faulting module name: KERNEL32.dll, version: 10.0.17763.1879, time stamp: 0x35e575b7
Exception code: 0xc00000fd
Fault offset: 0x000000000001096d
Faulting process id: 0xbda0
Faulting application start time: 0x01d76764f640faa0
Faulting application path: C:\Program Files\Tabular Editor 3\TabularEditor3.exe
Faulting module path: C:\Windows\System32\KERNEL32.dll
Report Id: 798551f7-1973-4f0a-8260-5b68c20d4617
Faulting package full name: 
Faulting package-relative application ID:

**Comment by otykier** [comment]
Could you locate the `%localappdata%\TabularEditor3\errorreport.txt` file after a crash, and either post it here or [mail it to me](mailto:daniel@tabulareditor.com) for further investigation? Thank you!

**Comment by MSmerdel** [comment]
[errorreport.txt](https://github.com/TabularEditor/TabularEditor3/files/6694817/errorreport.txt)

I have attached the file here

**Comment by otykier** [comment]
Thanks - unfortunately, that error report seems to come from an unrelated issue. Regarding the columns, it seems like Tabular Editor is crashing before it has a chance to write to the errorreport.txt file, making it very difficult to debug.

Any chance you can send me the Model.bim file? We're happy to fill out an NDA form if needed.

**Comment by MSmerdel** [comment]
I have sent you an email with a minimal model in which I can reproduce the issue.

**Comment by kevarnold972** [comment]
I am experiencing the same issue when editing a model directly in a PPU workspace. There is no errorreport file in my directory. I am creating measures I tested in DAX Studio. I was trying to create 3 measures before saving to the connected model. The 3rd measure is a copy/paste of the 2nd. When I changed my workflow to save between the 2nd and 3rd I could perform the copy/paste without a crash. 

Build 3.0.7838.17724
